### ISSUE TITLE
version 1.9.25 does not start

### ISSUE THREAD

**Issue by NektoNektovich** [Slion/Fulguris#613]
android 6.0
I had version 1.9.24 and it worked fine. updated to version 1.9.25 and now does not start, the system says that an error occurred in the fulguris application :(

**Comment by Slion** [comment]
I can reproduce the issue:
```
java.lang.NoSuchMethodError: No virtual method getDisplay()Landroid/view/Display; in class Landroid/content/Context; or its super classes (declaration of 'android.content.Context' appears in /system/framework/framework.jar)
at fulguris.extensions.ContextExtensionsKt$$ExternalSyntheticApiModelOutline0.m(SourceFile:3)
at kotlin.ResultKt.getConfigId(SourceFile:6)
at fulguris.activity.WebBrowserActivity.updateConfigurationSharedPreferences(SourceFile:1)
at fulguris.activity.WebBrowserActivity.onCreate(SourceFile:19)
at android.app.Activity.performCreate(Activity.java:6272)
at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1108)
at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2387)
at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2494)
at android.app.ActivityThread.access$900(ActivityThread.java:157)
at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1356)
at android.os.Handler.dispatchMessage(Handler.java:102)
at android.os.Looper.loop(Looper.java:148)
at android.app.ActivityThread.main(ActivityThread.java:5551)
at java.lang.reflect.Method.invoke(Native Method)
at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:730)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:620)
```

**Comment by Slion** [comment]
@NektoNektovich Should be fixed in v1.9.26 which is already available for download. Many thanks for reporting that issue.

**Comment by NektoNektovich** [comment]
thank you very much for the correction!

сб, 23 мар. 2024 г. в 21:26, Stéphane Lenclud ***@***.***>:
